reaktive/src/commonMain/kotlin/com/badoo/reaktive/maybe/Timer.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.reaktive.maybe
+
+import com.badoo.reaktive.scheduler.Scheduler
+
+fun maybeTimer(delayMillis: Long, scheduler: Scheduler): Maybe<Long> =
```

Comment:
Isn't it would be better to emit Unit? Same for other variants?

Reply:
It's a good question. It seems like other libraries (RxJava, RxJs, RxSwift) emits number. RxJava emits 0L, RxJs emits delay value. But Unit is also good choice.
I think delay value can be useful in complex data flows like this:
`
longObservable.flatMap { delay ->
    Observable.timer(delay, computationScheduler)
         .map(Long::toString)
         .startWith("Timer started")
}
`

Reply:
Also I think about replacing `delayMillis: Long` by time span data type. I was looking at [TimeSpan](https://github.com/korlibs/klock/blob/master/klock/src/commonMain/kotlin/com/soywiz/klock/TimeSpan.kt).
Time span data type instead of Long can prevent some kind of errors. But we should pay attention to backward  compatibility.

Reply:
Let's emit delay value. 0L is meaningless and useless. TimeSpan is a good thing but it's external dependency we don't have at the moment. We will consider it though. I'll create an issue.

Reply:
Fixed. Now timer operator emits delay value.